php/helpers.php

```diff
@@ -126,9 +137,14 @@ function block_row( $name ) {
  * @return bool
  */
 function block_rows( $name ) {
-	global $block_lab_attributes;
+	/**
+	 * An array containing the block attributes.
+	 *
+	 * @var array $attributes
+	 */
+	$attributes = block_lab()->loader->get_data( 'attributes' );
 
-	if ( ! isset( $block_lab_attributes[ $name ] ) ) {
+	if ( ! $attributes || ! isset( $attributes[ $name ] ) ) {
```

Comment:
`! isset( $attributes[ $name ] )` should be enough here, as it'll be false if `! isset( $attributes )`

Reply:
Done in https://github.com/getblocklab/block-lab/pull/435/commits/0fbf45adb15a4ce1a8fd10112939b68a67249365